docs/chaosiq-on-k8s-operator/settings-as-secret.md

```diff
@@ -0,0 +1,35 @@
+This section explains how to configure and use this operator for running
+ChaosIQ verifications and experiments.
+
+To be able to fetch assets from ChaosIQ, as well as pushing back results,
+for both experiments and verifications, you'll need to setup a settings file
+with a valid token and selected targeted organization & team.
+
+Please refer to these documentations, to [signin to ChaosIQ][SignInChaosIQ]
```

Comment:
Please refer to these documentations, - suggest changing this to
Please refer to these documentation links,